control/server.py

```diff
@@ -113,6 +113,13 @@ def serve(self):
 
         # Start server
         self.server.start()
+        enable_discovery_controller = self.config.getboolean("gateway", "enable_discovery_controller")
```

Comment:
```suggestion
        enable_discovery_controller = self.config.getboolean_with_default("gateway", "enable_discovery_controller", False)
```

If `enable_discovery_controller ` is not defined in conf file, default to False. Same applies to test conf value access